docs/src/main/sphinx/security/file-system-access-control.rst

```diff
@@ -106,6 +113,15 @@ DELETE FROM                          all                delete
 UPDATE                               all                update
 ==================================== ========== ======= ==================== ===================================================
 
+Permissions required for executing functions and PTFs (Polymorphic Table Function):
```

Comment:
```suggestion
Permissions required for executing functions:
```
I don't think we need to be specifically calling out PTFs - they're also functions, and while they're neat, they don't need super special treatment relative to other functions.

Reply:
Changed